src/Security/Security.php

```diff
@@ -1138,6 +1202,23 @@ public static function database_is_ready()
     }
 
     /**
+     * Resets the database_is_ready cache
+     */
+    public static function clear_database_is_ready()
+    {
+        self::$database_is_ready = null;
+        self::$force_database_is_ready = null;
```

Comment:
It's a testing helper. I already resent that it exists at all I don't want to make the API even more complex by having separate methods for this. I'll clarify behaviour in the docblock.